deploy/assignParcels.js

```diff
@@ -6,14 +6,15 @@ const {
   getConfiguration,
   readJSON,
   isEmptyObject,
-  waitForFailedTransactions,
+  waitForTransactions,
   isEmptyAddress
 } = require('./utils')
 
 const LANDRegistryArtifact = artifacts.require('LANDRegistry')
 const LANDRegistryDecorator = require('./LANDRegistryDecorator')
 
-const BATCH_SIZE = 50
+const LANDS_PER_ASSIGN = 50
+const BATCH_SIZE = 1
 const IGNORE_FAILED_TXS = true
 const REQUIRED_ARGS = ['parcels', 'account', 'password', 'owner']
```

Comment:
`password` arg should be optional to allow for not trying to unlock accounts without password